modules/core/src/lib/mode-handlers/translate-handler.js

```diff
@@ -0,0 +1,133 @@
+// @flow
+
+import turfBearing from '@turf/bearing';
+import turfDistance from '@turf/distance';
+import turfTransformTranslate from '@turf/transform-translate';
+import { point } from '@turf/helpers';
+import type { Geometry, Position } from '../../geojson-types.js';
+import type {
+  PointerMoveEvent,
+  StartDraggingEvent,
+  StopDraggingEvent,
+  DeckGLPick
+} from '../event-types.js';
+import type { EditAction } from './mode-handler.js';
+import { ModeHandler } from './mode-handler.js';
+
+export class TranslateHandler extends ModeHandler {
```

Comment:
nit: i see this is based on an old version of `RotateHandler`. Might be cleaner if `RotateHandler`, `ScaleHandler`, and `TranslateHandler` are all more closely aligned in their logic.